frontend/src/components/phones/dashboard/SendersPanelView.tsx

```diff
@@ -0,0 +1,186 @@
+        (a, b) =>
+          // Sort by last sent date
+          moment(b.last_inbound_date).unix() -
+          moment(a.last_inbound_date).unix()
+      )
+      .map((data) => {
+        return (
+          <li
+            key={data.id}
+            className={data.blocked ? styles["greyed-contact"] : ""}
+          >
+            <span className={styles["sender-number"]}>
+              {formatPhoneNumberToUSDisplay(data.inbound_number)}
+            </span>
+            <span
+              className={`${styles["sender-date"]} ${styles["sender-date-wrapper"]}`}
+            >
+              {data.last_inbound_type === "text" && (
+                <ForwardedTextIcon
+                  alt="Last received a text"
+                  className={styles["forwarded-type-icon"]}
+                  width={20}
+                  height={12}
+                />
+              )}
+              {data.last_inbound_type === "call" && (
+                <ForwardedCallIcon
+                  alt="Last received a call"
+                  className={styles["forwarded-type-icon"]}
+                  width={20}
+                  height={15}
+                />
+              )}
+              <Moment calendar={calendarStrings}>
+                {data.last_inbound_date}
+              </Moment>
+            </span>
+            <span className={styles["sender-controls"]}>
+              <button
+                onClick={() =>
+                  inboundContactData.setForwardingState(!data.blocked, data.id)
+                }
+                className={styles["block-btn"]}
+              >
+                {data.blocked ? "Unblock" : "Block"}
+              </button>
+            </span>
+          </li>
+        );
+      });
+
+  const senderLogsPanel = (
+    <ul className={styles["caller-sms-senders-table"]}>
+      <li className={styles["greyed-contact"]}>
+        <span>
+          {l10n.getString("phone-dashboard-sender-table-title-sender")}
+        </span>
+        <span>
+          {l10n.getString("phone-dashboard-sender-table-title-activity")}
+        </span>
+        <span>
+          {l10n.getString("phone-dashboard-sender-table-title-action")}
+        </span>
+      </li>
+      {inboundContactArray}
+    </ul>
+  );
+
+  return (
+    <div id="secondary-panel" className={styles["dashboard-card"]}>
+      <div className={styles["dashboard-card-caller-sms-senders-header"]}>
+        <span>
+          <button
+            type="button"
+            onClick={() => props.back_btn()}
+            className={styles["caller-sms-logs-back-btn"]}
+          >
+            <ChevronLeftIcon
+              alt="Back to Primary Dashboard"
+              className={styles["nav-icon"]}
+              width={20}
+              height={20}
+            />
+          </button>
+        </span>
+        <span className={styles["caller-sms-logs-title"]}>
+          {l10n.getString("phone-dashboard-senders-header")}
+        </span>
+        <span></span>
```

Comment:
question (non-blocking): What is this empty span for? Should we add a comment so we know what will go into it?

Reply:
It's actually just a way to format the header